package.json

```diff
@@ -29,15 +31,28 @@
   },
   "homepage": "https://github.com/rgbkrk/pipboy#readme",
   "devDependencies": {
-    "babel-preset-es2015": "^6.1.18",
-    "electron-compile": "^1.0.1",
-    "electron-compilers": "^1.0.1",
+    "babel-core": "^5.8.34",
+    "babel-loader": "^5.4.0",
```

Comment:
Well there is the possibility of adding loaders for icons (svg), map tiles (png), stylesheets, the html, etc. So let's not remove webpack just yet ;)